debian/mu.install

```diff
@@ -1,2 +1,2 @@
-conf/mu.desktop usr/share/applications/
-conf/mu.png usr/share/icons/hicolor/256x256/apps/
+conf/mu.codewith.editor usr/share/applications/
```

Comment:
Well that's embarrassing, missed the .desktop here

Oops

Reply:
What should it be and I'll update now on master.

Reply:
`conf/mu.codewith.editor usr/share/applications/` -> `conf/mu.codewith.editor.desktop usr/share/applications/`

Reply:
+1